app/scripts/modules/amazon/src/function/configure/FunctionDebugAndErrorHandling.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+
+import {
+  FormikFormField,
+  IWizardPageComponent,
+  HelpField,
+  TextInput,
+  ReactSelectInput,
+  ValidationMessage,
+} from '@spinnaker/core';
+import { FormikProps, FormikErrors } from 'formik';
+import { IAmazonFunctionUpsertCommand } from 'amazon/index';
+import { IAmazonFunction } from 'amazon/domain';
+
+export interface IFunctionDebugAndErrorHandlingProps {
+  formik: FormikProps<IAmazonFunctionUpsertCommand>;
+  isNew?: boolean;
+  functionDef: IAmazonFunction;
+}
+
+export interface IFunctionDebugAndErrorHandlingState {
+  some: string;
+}
+
+export class FunctionDebugAndErrorHandling
+  extends React.Component<IFunctionDebugAndErrorHandlingProps, IFunctionDebugAndErrorHandlingState>
+  implements IWizardPageComponent<IAmazonFunctionUpsertCommand> {
+  constructor(props: IFunctionDebugAndErrorHandlingProps) {
+    super(props);
+    this.state = { some: '' };
+  }
+  public validate = (values: IAmazonFunctionUpsertCommand): FormikErrors<IAmazonFunctionUpsertCommand> => {
+    const errors = {} as any;
```

Comment:
could also migrate this to `ValidateForm` api.
If we do migrate to ValidateForm, then all these AWS specific validators (for ARN, IAM Role, etc) could be placed in a file called `aws.validators.ts` and reused elsewhere.

Reply:
Done. change will appear in the next push